### PR TITLE
Refactored code from using momentjs to date-fns.

### PR DESCRIPTION
**Pull request by thaigillespie**
Removed moment and @date-io/moment (this one is for
Material-UI pickers) from the codebase.

date-fns isn't too bad to use : )

### CODE REVIEW

File: src/pages/PointsPage/point_display.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ class PointDisplay extends React.Component {
           <div>
             <Grid container spacing={2}>
               {points.map(event => {
+                console.log(event.date);
```
Comment (godwinpang) [inline src/pages/PointsPage/point_display.js:52]: hehe

**Comment by godwinpang** [comment]
if you're looking to reduce build bundle size we should fix treeshaking for mui and see where all the other stupid stuff is coming from

File: src/pages/PointsPage/point_display.js (hunk 1 above)
Comment (thaigillespie) [reply src/pages/PointsPage/point_display.js:52]: lolol rip :)) I ended up figuring out its type thru something else, I'll fix this.

**Comment by thaigillespie** [comment]
How do we go about fixing treeshaking for mui? I don't think we do the treeshaking on our own, right?

**Comment by godwinpang** [comment]
dw about it - we can tackle that later

**Comment by thaigillespie** [comment]
Okay reverted version of react-scripts to 3.4.0 from ^3.4.2 and circleci works again. This should be good to go.

**Comment by godwinpang** [comment]
next time tag me so i get a notif

**Comment by thaigillespie** [comment]
Will do, mbmb

**Pull request merged**